launcher/settingsView/csettingsview_moc.cpp

```diff
@@ -40,6 +40,14 @@ static const std::string cursorTypesList[] =
 	"software"
 };
 
+static const std::string autosaveModeList[] =
+{
+	"Off",
+	"Using Counter",
+	"Using Turn Info",
+	"Using Schema"
+};
+
```

Comment:
This strings should be added directly into widget to make them translateable

Reply:
using now custom data for controls:

```	ui->comboBoxAutosaveMode->setItemData(0, "Off", Qt::UserRole);
	ui->comboBoxAutosaveMode->setItemData(1, "Using Counter", Qt::UserRole);
	ui->comboBoxAutosaveMode->setItemData(2, "Using Turn Info", Qt::UserRole);
	ui->comboBoxAutosaveMode->setItemData(3, "Using Schema", Qt::UserRole);
```